src/components/PageHeader/_PageHeader.scss

```diff
@@ -99,3 +100,27 @@
     }
   }
 }
+
+.coa-PageHeader--search {
+  padding-top: $coa-spacing-large;
+  padding-bottom: $coa-space-level3;
+  margin-bottom: $coa-spacing-md-small;
+  background: $coa-color-secondary;
+  h1 {
+    padding-top: $coa-spacing-xsmall;
+    font-size: $coa-font-size-medium;
+    text-align: center;
+  }
+  @include media($coa-medium-screen) {
+    h1 {
+      text-align: left;
+      font-size: $coa-font-size-xxlarge;
+    }
+  }
+  @include media($coa-small-screen) {
```

Comment:
`@include mobile {` should work for this line

Reply:
after lookin' into this I realized the designs were specific to sub 480px. So, I went ahead and created a mobile-small in the _mixins.scss.